src/components/UsedByRow.tsx

```diff
@@ -0,0 +1,89 @@
+import { type FC } from "react";
+import UsedByItem from "./UsedByItem";
+import ExpandableList from "./ExpandableList";
+import { filterUsedByType } from "util/usedBy";
+import { pluralize } from "util/instanceBulkActions";
+import type { ResourceIconType } from "components/ResourceIcon";
+import { capitalizeFirstLetter } from "util/helpers";
+import { linkForVolumeDetail } from "util/storageVolume";
+import type { LxdStorageVolume } from "types/storage";
+import { getStorageBucketURL } from "util/storageBucket";
+import ResourceIcon from "components/ResourceIcon";
+import { useCurrentProject } from "context/useCurrentProject";
+import type { ResourceType } from "util/resourceDetails";
+
+interface Props {
+  entityType: ResourceType;
+  usedBy: string[] | undefined;
+}
+
+const UsedByRow: FC<Props> = ({ entityType, usedBy }) => {
+  if (!usedBy || usedBy.length === 0) return null;
+
+  const { project } = useCurrentProject();
+
+  const lxdUsedBy = filterUsedByType(entityType, usedBy);
+
+  if (lxdUsedBy.length === 0) return null;
+
+  let entityName = pluralize(
+    capitalizeFirstLetter(entityType.replaceAll("-", " ")),
+    lxdUsedBy.length,
+  );
+
+  if (entityType === "volume") {
+    entityName = pluralize("Custom volume", lxdUsedBy.length);
+  }
+
+  return (
+    <tr>
```

Comment:
I think we should display a thin line between the rows if there are more than one, something like this:

```
.used-by-row:not(:first-child) {
    border-top: $border-thin;
}
```
```suggestion
    <tr classname="used-by-row">
```

I think the lines improve especially very busy pages like pools:

<img width="1917" height="909" alt="image" src="https://github.com/user-attachments/assets/6caba89e-d874-4dcd-aa00-f4839c4b56e3" />